runtime/onert/api/nnapi/wrapper/ANeuralNetworksExecution.cc

```diff
@@ -136,7 +136,7 @@ bool ANeuralNetworksExecution::setInput(uint32_t index, const ANeuralNetworksOpe
   try
   {
     onert::ir::IOIndex input_index{index};
-    const auto shape =
+    const auto &shape =
       (type != nullptr) ? NNAPIConvert::getShape(type) : _execution->getInputShape(input_index);
```

Comment:
`NNAPIConvert::getShape()` can be optimized by RVO. Adding reference actually interferes with RVO. 
`getInputShape()`'s return type is value type and the returned object's type is a lvalue reference. So, `getInputShape()` always calls it own copy constructor when it is called.